vars/piperStageWrapper.groovy

```diff
@@ -80,7 +80,7 @@ private void executeStage(script, originalStage, stageName, config, utils, telem
         def body = originalStage
 
         // First, check if a global extension exists via a dedicated repository
-        if (globalExtensions) {
+        if (!Boolean.valueOf(env.PIPER_NO_EXTENSIONS) && globalExtensions) {
```

Comment:
Could there be cases where we want to have global but no local extension? Maybe introducing an additional flag would be better but could also be added later on if needed.

Reply:
For external usecase, none of them are supported currently. Is it fine if we add that change in later point of time?